### DISCUSSION TITLE
Missing Awesome Fonts

### DISCUSSION THREAD

**Discussion by Y3510X** [react-icons/react-icons#614] [General]
Although the react-icons documentation states that FontAwesome v5.15.4 fonts are included in react-icons, it appears some fonts are missing.  An example, fa-album-collection, according to the FontAwesome documentation, is part of v5.15.4 but is missing from react-icons.  

fa-album-collection as shown on v5.15.4
https://fontawesome.com/v5/search?q=album&o=r&f=brands%2Cclassic

Missing from react-icons documentation:
https://react-icons.github.io/react-icons/icons?name=fa

**Comment by si458** [comment]
this is showing as a PRO icon so it wont be included as only the free ones are included
![image](https://github.com/react-icons/react-icons/assets/765314/951090c1-6ff5-495d-aeb3-66493e660b15)
